inc/az_ulib_ipc_api.h

```diff
@@ -150,6 +172,41 @@ AZ_NODISCARD az_result az_ulib_ipc_publish(
     const az_ulib_interface_descriptor* const interface_descriptor,
     az_ulib_ipc_interface_handle* interface_handle);
 
+/**
+ * @brief   Make an interface default in the device.
```

Comment:
>Make an interface default in the device. [](http://example.com/codeflow?start=12&length=40)

do you mean something more along the lines of "Set a default package for a given interface."?
This wording makes it seem like you're setting a default interface version.